docs/usage/permissions.rst

```diff
@@ -0,0 +1,55 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+Access control
+--------------
+
+Invenio-Files-Rest depends on `Invenio-Access
+<https://invenio-access.readthedocs.io>`_ module, to control the files access.
+
+It comes with a default permission factory implementation which can be found
+at :code:`invenio_files_rest.permissions.permission_factory` and can be
+customized further, by providing our custom implementation in the relevant
+config variable :code:`FILES_REST_PERMISSION_FACTORY`.
+
```

Comment:
Some suggestions:
`
It provides a default implementation of permissions for several different actions, defined by (`path`), but it also allows to provide your own custom implementation by using the config var <name>
`